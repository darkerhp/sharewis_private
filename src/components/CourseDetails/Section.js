import React from 'react';
import ReactNative from 'react-native';

const { Component, PropTypes } = React;
const { View, StyleSheet, Text, Slider, Dimensions } = ReactNative;

const styles = StyleSheet.create({
  container: {
    backgroundColor: '#F5F5F5',
    justifyContent: 'flex-start',
    alignItems: 'flex-start',
    flexDirection: 'row',
    padding: 7,
    borderColor: '#DDDDDD',
    borderBottomWidth: 1,
  },
  sectionTitleText: {
    fontSize: 14,
    color: '#525252',
  },
});

const Section = ({ lecture }) =>
  <View style={styles.container}>
    <Text style={styles.sectionTitleText}>{lecture.title}</Text>
  </View>;


Section.propTypes = {
  lecture: PropTypes.object.isRequired
};

export default Section;
