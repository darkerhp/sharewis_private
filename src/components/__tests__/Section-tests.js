/* eslint-disable no-undef */
/* eslint-disable import/no-extraneous-dependencies */
import 'react-native';
import renderer from 'react-test-renderer';
import React from 'react';
import Section from '../CourseDetails/Section';

describe('Section', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Section
          lecture={{
            title: 'Title'
          }}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
