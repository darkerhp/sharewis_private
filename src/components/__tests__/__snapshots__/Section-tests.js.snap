// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-start",
      "backgroundColor": "#F5F5F5",
      "borderBottomWidth": 1,
      "borderColor": "#ddd",
      "flexDirection": "row",
      "justifyContent": "flex-start",
      "padding": 7,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#4a4a4a",
        "fontSize": 14,
      }
    }
  >
    Title
  </Text>
</View>
`;
